docs/design-guides/architecture-overview.md

```diff
@@ -0,0 +1,49 @@
+# Architecture Design
+
+## Table of content
+
+- [Overview](#overview)
+- [Components](#components)
+  - [Motivation for leveraging Graph Open Extensions for Identity Mapping](#motivation-for-leveraging-graph-open-extensions-for-identity-mapping)
+- [Limitations](#limitations)
+- [Guidance](#guidance)
+- [Other Helpful Links to Explore](#other-helpful-links-to-explore)
+
+## Overview
+This sample solution is primarily focused on building the ACS Authentication Backend Server backed by Azure Active Directory as the Identity Provider and using open source libraries from [Microsoft Identity Platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview). The sample solution can be used directly in below scenarios, otherwise the sample needs to be adapted as described in Guidance section below:
```

Comment:
```suggestion
This sample is primarily focused on building a Trusted Service for Azure Communication Services authentication. It is backed by Azure Active Directory as the Identity Provider and using open source libraries from [Microsoft Identity Platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview). The sample can be used directly in below scenarios, otherwise the sample needs to be adapted as described in Guidance section below:
```